src/System.Net.HttpListener/src/System/Net/HttpResponseStream.cs

```diff
@@ -40,6 +40,19 @@ public override IAsyncResult BeginRead(byte[] buffer, int offset, int size, Asyn
         }
 
         public override int EndRead(IAsyncResult asyncResult) => throw new InvalidOperationException(SR.net_writeonlystream);
+        
+        public new Task<int> ReadAsync(Byte[] buffer, int offset, int count)
```

Comment:
Here I used `new` instead of `virtual`, because compiler will complain:
1. 'HttpResponseStream.ReadAsync(byte[], int, int)' is a new virtual member in sealed class 'HttpResponseStream'
2. 'HttpResponseStream.ReadAsync(byte[], int, int)' hides inherited member 'Stream.ReadAsync(byte[], int, int)'. Use the new keyword if hiding was intended.